### DISCUSSION TITLE
Defining a problem matcher without `"file"`

### DISCUSSION THREAD

**Discussion by tinder-tannerbennett** [microsoft/vscode-discussions#1140] [Extension Development QnA] [answered]
Is there a way to define a problem matcher that doesn't know about / care about any file in particular? I want problems with tasks that run in my extension to appear in the problems pane, but they don't always correspond to a specific file. For example, Bazel will sometimes output an error like
```
ERROR: An error occurred during the fetch of repository 'Foo'
```
And I just want to list the error in the problems pane without associating the error with a file.

**Comment by tamuratak** [comment, marked as answer]
It doesn't seem supported. See https://github.com/microsoft/vscode/issues/155673
